browser/test/integration/uiSlotValues.js

```diff
@@ -0,0 +1,68 @@
+var assert = require('assert');
```

Comment:
but it doesn't? this sets it to call update_input() with the overridden values on initialise()